main.go

```diff
@@ -170,6 +180,7 @@ func main() {
 	if err := secretReconciler.RemoveInvalidAnnotationsFromLinuxNodes(mgr.GetConfig()); err != nil {
 		setupLog.Error(err, "error removing invalid annotations from Linux nodes")
 	}
+	//+kubebuilder:scaffold:builder
```

Comment:
Why is this needed?

Reply:
For my own education; is this just a comment or lines like this one get parsed elsewhere (annotations maybe)?

cc: @sebsoto  @aravindhp

Reply:
How about adding a comment for our benefit?

Reply:
> This is poorly documented by OperatorSDK/kubebuilder, Theres [an issue open](https://github.com/kubernetes->sigs/kubebuilder/issues/1487) to increase documentation around it.
> 

@sebsoto  thanks for adding more color here. I found [this comment](https://github.com/kubernetes-sigs/kubebuilder/issues/1487#issuecomment-823864494)   very helpful.